R/metrics.R

```diff
@@ -430,6 +434,8 @@ make_numeric_metric_function <- function(fns) {
     # Construct calls from the functions + arguments
     calls <- lapply(fns, call2, !!! call_args)
 
+    calls <- lapply(calls, call_remove_duplicate_arguments)
+
     # Evaluate
     metric_list <- mapply(
       FUN = eval_safely,
```

Comment:
Just putting this somewhere:

At the end of the `@details` section of `metric_set()`, after it talks about the function signatures that it generates, you may want to mention something like:

> If `metric_tweak()` has been used to "tweak" one of these arguments, like `estimator` or `event_level`, then the tweaked version wins. This allows you to set the `estimator` on a metric by metric basis if needed.